csharp/sources/Valkey.Glide/Pipeline/IBatch.cs

```diff
@@ -33,4 +33,88 @@ internal interface IBatch
     /// <inheritdoc cref="IServerManagementCommands.Info(Section[])" path="/param" />
     /// <returns>Command Response - <inheritdoc cref="IServerManagementCommands.Info(Section[])" /></returns>
     IBatch Info(Section[] sections);
+
```

Comment:
These should go in their own IBatchSetCommand.cs to avoid merge conflicts.  We want to avoid having files with 300 methods in them. IBatch should then inherit from IBatchSetCommand along with other BatchCommand interfaces.